packages/core/src/fhirpath/atoms.test.ts

```diff
@@ -52,6 +52,20 @@ describe('Atoms', () => {
     expect(evalFhirPath('x | y', [])).toEqual([]);
   });
 
+  test.each([
+    ['true implies true', true],
+    ['true implies false', false],
+    ['true implies null', undefined],
```

Comment:
Can you add test cases for the "empty" column on this page: https://build.fhir.org/ig/HL7/FHIRPath/

You might need to refactor the `test.each()` array to be able to handle the empty array case.

```ts
['true implies {}', []],
['false implies {}', [true]],
['{} implies false', []],
['{} implies {}', []],
```